test-lit/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2018-2019, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Why is this Nvidia copyright?
Also, do you want to use the new licensing here?

Reply:
The problem of C&P

Reply:
LLVM convention has removed the requirement for copyright.  @gklimowicz is working on updating the f18 files with the new header.

Reply:
It looks that in llvm CMakeLists.txt does not have a header like the tests.

Reply:
Done!